README.md

```diff
@@ -447,6 +447,16 @@ $ REPO_ROOT=`pwd` ginkgo '-focus=sanity' -failFast ./test/e2e/
 ...
 ```
 
+#### Running E2E tests in DeviceMode:Direct
+
+Modify two test scripts to use deployment in Direct mode:
+
+```sh
+    $ sed -i 's/csi.yaml/csi-direct.yaml/' test/e2e/storage/sanity.go test/start-stop.make
```

Comment:
how about changing the link, instead?

Reply:
I agree, changing link is cleaner than patching testing files. For that, we need to create link for unified file as well. I will do that.

Reply:
Both approaches have the problem that they need to modify content of the git repo. A better solution would be to make sanity.go and start-stop.make configurable such that they do the right thing when an env variable is set. Env variables (in contrast to command line variables) can be checked both by make and Go code.

Reply:
> Both approaches have the problem that they need to modify content of the git repo. A better solution would be to make sanity.go and start-stop.make configurable such that they do the right thing when an env variable is set. Env variables (in contrast to command line variables) can be checked both by make and Go code.

I agree.

Reply:
I agree as well, modifying state of repo objects is messy way, so let's avoid it. I will try to introduce env.vars, with default leading to LVM mode.